compositor_render/src/transformations/builtin/fixed_position_layout.wgsl

```diff
@@ -0,0 +1,90 @@
+struct VertexInput {
+    @location(0) position: vec3<f32>,
+    @location(1) tex_coords: vec2<f32>,
+    @location(2) texture_id: i32,
```

Comment:
What will happen if this transform does not have any input?

Reply:
I forgot to handle that :)